### ISSUE TITLE
Frontmatter metadata are not working correctly

### ISSUE THREAD

**Issue by Slashgear** [thepracticaldev/dev.to#7973]
**Describe the bug**

Some of metadata I put in frontmatter part seems not to work as described.
- description is missing
- canonical is mission

**To Reproduce**

Go to https://dev.to/slashgear_/discover-jest-hidden-feature-automock-2c9f

Look at page metadata (you could use meta debugger devtools extension like this one https://chrome.google.com/webstore/detail/meta-debugger/jfpdemgdamgplelnlmaecbonkfgfgomp)

You should not see description or canonical

**Expected behavior**

All frontmatter metadatas should work correctly

**Screenshots**
<img width="1129" alt="Capture d’écran 2020-05-20 à 08 09 51" src="https://user-images.githubusercontent.com/6263857/82411053-4fdbac80-9a71-11ea-89db-516a7a10761c.png">

**Desktop (please complete the following information):**

- OS: Macos latest
- Browser: Chrome
- Version: 81

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

**Comment by Ridhwana** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled "type: bug" are always up for grabs, but for feature requests, please wait until we add a ready for dev before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our contributors' guide.

**Comment by luchiago** [comment]
Hi, can I work on this?

**Comment by rhymes** [comment]
Sure @luchiago, I'm going to assign you to the issue. Thanks!

**Comment by luchiago** [comment]
@rhymes I tested with the [debugger extension](https://chrome.google.com/webstore/detail/meta-debugger/jfpdemgdamgplelnlmaecbonkfgfgomp)  in other website and worked. I compared the source code of this article page and the other page and both exist the `<link rel="canonical">` and `<meta name="description">`. Do you think it can be related to the extension used for debugging?
![image](https://user-images.githubusercontent.com/30028621/82969552-50f76700-9fa6-11ea-92d1-0596a5b39022.png)

**Comment by atsmith813** [comment]
The [debugger extension](https://chrome.google.com/webstore/detail/meta-debugger/jfpdemgdamgplelnlmaecbonkfgfgomp) looks [only at`head` tags](https://github.com/bluemoehre/chrome-meta-debugger/blob/1b2f92c3cfcf4b0dae9eaa71ab007a3dcfd42344/src/js/devtools.js#L39) and we are rendering certain metadata (i.e. description and canonical) outside of the `head` tag.

I'll defer to @rhymes if this is an issue for us to be doing that or not.

cc: @luchiago @Slashgear

**Comment by rhymes** [comment]
@luchiago I'm not sure what it's supposed to happen as I'm not familiar with this extension

what should we do to fix this?

**Comment by atsmith813** [comment]
Closing this out, thanks @luchiago for pointing out that we are rendering the metadata!

To clarify, we are rendering the metadata - we just do it outside of the `head` tag which is the only place the Chrome add-on looks, so it doesn't show up in the add-on view. This isn't an issue.

**Comment by luchiago** [comment]
@atsmith813 Okay. @rhymes we don't need to fix as the tags are being rendered correctly.

**Comment by luchiago** [comment]
@atsmith813 I was wondering if those tags should be inside the head tag, because I'm not sure if search engines as Google will look for this tags outside the head tag

**Comment by atsmith813** [comment]
@luchiago they do look outside of the head! Technically this pattern of rendering them outside of the `head` isn't recommended practice, but it works.

I have a feeling we did that in the past intentionally, but I'm not sure why. I'll ask around on the team to find out. If I don't come up with anything, I'll open a separate issue to move the tags.

**Comment by atsmith813** [comment]
Actually, it seems it's because we're using article-specific content in those tags is why we're doing it that way!

**Comment by luchiago** [comment]
@atsmith813 Humm, now I get it.
